toolchain/fortify-headers/patches/002-strings.h-add-__extension__-mark.patch

```diff
@@ -0,0 +1,12 @@
+--- a/include/strings.h
```

Comment:
Please add a description to this patch file too. For example the content of the OpenWrt patch description.  You can generate it for git format-patch.

Reply:
Done, I also rebased on the latest main branch.

I noticed that when the previous patch was added to fortify-headers the `PKG_VERSION` was updated, should I do that as well?

Reply:
thanks for the update.

`PKG_VERSION` is used to trigger an update of the package by opkg, this code is not updated by opkg so it is not needed.

Reply:
thanks for the update.

`PKG_VERSION` is used to trigger an update of the package by opkg, this code is not updated by opkg so it is not needed.